### ISSUE TITLE
Build phase script error with output paths 

### ISSUE THREAD

**Issue by ayushgoel** [Karumi/Kin#34]
Similar to https://github.com/Karumi/Kin/issues/18

```
ERROR: line 391:4 mismatched input '"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/DataModelBundle.bundle"' expecting {')', NON_QUOTED_STRING}
ERROR: line 428:4 mismatched input '"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/DataModelBundle.bundle"' expecting {')', NON_QUOTED_STRING}
ERROR: line 463:4 mismatched input '"$(DERIVED_FILE_DIR)/Pods-ModelKit_Tests-checkManifestLockResult.txt"' expecting {')', NON_QUOTED_STRING}
ERROR: line 481:4 mismatched input '"$(DERIVED_FILE_DIR)/Pods-MigrationTests-checkManifestLockResult.txt"' expecting {')', NON_QUOTED_STRING}
```

**Comment by Serchinastico** [comment]
Hi @ayushgoel 

Is it possible for you to send us your pbxproj file so that we can handle those scenarios? If not the whole file, I think it'd be enough with just those failing lines (391, 428, 463, 481) with some context (+-10 lines).

Thank you!

**Comment by chrise86** [comment]
@ayushgoel did you resolve your issue after?

**Comment by ayushgoel** [comment]
Hey sorry for not updating earlier. Here is the new output:

```
ERROR: line 7685:4 mismatched input '"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/XXX.bundle"' expecting {')', NON_QUOTED_STRING}
```

Excerpts from the pbxproj:

```
...
File: /Users/ayushgoel/code/Pingpong-iOS/Talkto.xcodeproj/project.pbxproj
7672: 		2DF538536D28468BE57AB16C /* [CP] Copy Pods Resources */ = {
7673: 			isa = PBXShellScriptBuildPhase;
7674: 			buildActionMask = 2147483647;
7675: 			files = (
7676: 			);
7677: 			inputPaths = (
7678: 				"${SRCROOT}/Pods/Target Support Files/Pods-CommonPods-XXX/Pods-CommonPods-XXX-resources.sh",
7679: 				"${PODS_CONFIGURATION_BUILD_DIR}/XXX/XXX.bundle",
7680: 				"${PODS_CONFIGURATION_BUILD_DIR}/XXX/XXX.bundle",
7681: 				"${PODS_ROOT}/XXX/Support/XXX.bundle",
7682: 			);
7683: 			name = "[CP] Copy Pods Resources";
7684: 			outputPaths = (
7685: 				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/XXX.bundle",
7686: 				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/XXX.bundle",
7687: 				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/XXX.bundle",
7688: 			);
7689: 			runOnlyForDeploymentPostprocessing = 0;
7690: 			shellPath = /bin/sh;
7691: 			shellScript = "\"${SRCROOT}/Pods/Target Support Files/Pods-CommonPods-XXX/Pods-CommonPods-XXX-resources.sh\"\n";
7692: 			showEnvVarsInLog = 0;
7693: 		};
7694: 		3A1F9A8418BF13CC00F87000 /* Copy Pods Acknowledgements plist */ = {
...
```

**This part of pbxproj is generated by cocoapods (1.4.0). In case you suspect that the error might be with generation itself, I can try again with updating cocoapods version.**

Note: I have concealed certain strings, do let me know in case that hinders with debugging, I will conceal better.

**Comment by Serchinastico** [comment]
Released a fix in Kin 1.7.2!
